edgecast/waf/bot_rule_models.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 Edgecast Inc., Licensed under the terms of the Apache 2.0
+// license. See LICENSE file in project root for terms.
+
+package waf
+
+// BotRule is a detailed representation of a Bot Manager Rule
+type BotRule struct {
+	/*
```

Comment:
We generally use line comments within the struct when defining fields. Should we discuss using block comments further as a team?

Reply:
I use block comments for lenghty descriptions. They seem to allow some basic formatting for intellisense as well.

Reply:
This was an incorrect statement about formatting. The comments have been converted to line comments.